zebrad/tests/acceptance.rs

```diff
@@ -45,6 +45,11 @@ use zebrad::config::ZebradConfig;
 /// metrics or tracing test failures in Windows CI.
 const LAUNCH_DELAY: Duration = Duration::from_secs(10);
 
+/// A regular expression that matches `zebrad` versions.
+///
+/// `zebrad` uses [semantic versioning](https://semver.org/).
+const ZEBRAD_VERSION_REGEX: &str = r"zebrad [0-9].[0-9].[0-9]-[A-Za-z]*.[0-9]+";
```

Comment:
So this allows "zebrad 9.9.9-.9" ?

Reply:
The regex is a temporary workaround,  see #2252 for a long-term fix.

Reply:
Also see my comment at https://github.com/ZcashFoundation/zebra/pull/2252#discussion_r645281202

Reply:
`.` in a regex doesn't mean the literal ASCII 0x2E dot character. It means any character. Use escaping or `[.]` for dot, but do versions always have 3 dots?

Reply:
> `.` in a regex doesn't mean the literal ASCII 0x2E dot character. It means any character. Use escaping or `[.]` for dot, but do versions always have 3 dots?

Thanks, but this regex has always been wrong, this PR is a temporary workaround until we merge #2252.